core/src/main/java/org/radargun/stages/test/TestStage.java

```diff
@@ -67,6 +67,9 @@
    @Property(doc = "Whether an error from transaction commit/rollback should be logged as error. Default is true.")
    public boolean logTransactionExceptions = true;
 
+   @Property(doc = "Operation per second.")
+   private int opsPerSec = 0;
```

Comment:
User should not calculate this by hand; you already have duration, thread number and delayBetweenRequests (which is incorrectly named as it's a cycle-time - period between starting requests)

Reply:
My intention here with `opsPerSec` is:
If it is 0, RadarGun will have the same behavior that we have today.
If it is greater than 0, RadarGun will start reporting in the response histogram. For sure, who will implement the benchmark file will need to start doing the formula that you mentioned by hand.

For the next PR, I was thinking about remove `thread` and `delayBetweenRequests` and just keep one thread using `CompletableFuture`.

Now that RadarGun have the notion of schedule ( intention of this PR), we know when the request should be started and when it was completed. This still pending about an answer in https://github.com/radargun/radargun/pull/545#discussion_r191424876

Reply:
Now we have only one histogram
```java
      if (intendedTimeNs != 0) { // if intended time is set we can compute response/wait
         histogram.recordValue(endTimeNs - intendedTimeNs);
      } else {
         histogram.recordValue(endTimeNs - startTimeNs);
      }
```

Reply:
So is there any reason to calculate this by hand? If you want just report intended start time instead of actual start time (ok for me), this could be a boolean property.